app/merginapi.cpp

```diff
@@ -706,15 +703,18 @@ void MerginApi::downloadProjectReplyFinished()
   QNetworkReply *r = qobject_cast<QNetworkReply *>( sender() );
   Q_ASSERT( r );
 
-  QString projectName = mPendingRequests.value( r->url() );
+  QString projectFullName = mPendingRequests.value( r->url() );
+  QStringList parts = projectFullName.split( "/" );
+  QString projectNamespace = parts.at( 0 );
+  QString projectName = parts.at( 1 );
```

Comment:
let's also put the namespace/project splitting code into a function - it should also return bool whether splitting was successful, the individual namespace and project names could be passed out through QString&